HDF5Examples/C/H5T/h5ex_t_opaqueatt.c

```diff
@@ -121,7 +121,7 @@ main(void)
      */
     printf("Datatype tag for %s is: \"%s\"\n", ATTRIBUTE, tag);
     for (i = 0; i < dims[0]; i++) {
-        printf("%s[%llu]: ", ATTRIBUTE, i);
+        printf("%s[%" PRIuHSIZE "]: ", ATTRIBUTE, i);
```

Comment:
is PRIuHSIZE available afterwards when using a binary to build the examples?

Reply:
Yes, It's in H5public.h.